alphadia/workflow/peptidecentric.py

```diff
@@ -557,40 +399,22 @@ def calibration(self):
                 verbosity="progress",
             )
             return
-        self.timing_manager.start("optimization")
+        self.timing_manager.set_start_time("optimization")
```

Comment:
shall we move the timing logic one level up to the calling code? (also for "extraction")